Userland/Applications/VideoPlayer/VideoPlayerWidget.h

```diff
@@ -43,8 +43,8 @@ class VideoPlayerWidget final : public GUI::Widget {
     ErrorOr<void> setup_interface();
     void update_play_pause_icon();
     void update_seek_slider_max();
-    void set_current_timestamp(Time);
-    void set_time_label(Time);
+    void set_current_timestamp(Duration);
+    void set_time_label(Duration);
```

Comment:
those two look too suspicious for durations…

Reply:
Exactly, further candidates for future changes.